README.md

```diff
@@ -13,28 +13,34 @@ Include = /etc/pacman.d/mirrorlist
 ```
 
 Then install the following dependencies:  
-`sudo pacman -S --needed unzip sed xdelta3 jre8-openjdk lib32-libpulse`
-
+`sudo pacman -S --needed unzip sed xdelta3 patchelf jre8-openjdk lib32-libpulse`
 
 ### Debian (including Ubuntu, Mint, PopOS, etc.)
 Make sure that the i386 architecture is enabled for you, as libopenal1 requires the 32-bit version, and Ubuntu does not enable 32-bit support by default.  
 To enable it, do the following:
 ```
 sudo dpkg --add-architecture i386
-sudo apt update && sudo apt install libc6:i386 libncurses5:i386 libstdc++6:i386
+sudo apt update && sudo apt install libc6:i386 libncurses5:i386 libstdc++6:i386 libopenal1:i386
 ```
 
 Then install the following dependencies:  
-`sudo apt install unzip sed xdelta3 openjdk-8-jre libopenal1:i386`
+`sudo apt install unzip sed xdelta3 patchelf openjdk-8-jre`
+
+### Nix (Including NixOS)
+Enter a shell with all the required dependencies to run the script:
+`nix shell nixpkgs#{gnused,unzip,xdelta,patchelf,jre}`
+
+After running the script the game can be started using the `steam-run` FHS environment, as to avoid having to patch its dynamic linker and dynamically linked libraries. You can run it with the following command:
+`NIXPKGS_ALLOW_UNFREE=1 nix shell --impure nixpkgs#steam-run --command steam-run ./runner`
```

Comment:
> Also, since the section at the top if for running the generated AppImage, are there any differences between running that, and running the "pure" binary

The appimage does not work on NixOS due to outdated/insecure dependencies. We prefer not to use them anways. I put in a notice.

> If yes, then those should probably be also clarified at the bottom under the "manual installation" section.

Done :+1: